src/Microsoft.Identity.Web/TokenCacheProviders/Distributed/MsalDistributedTokenCacheAdapter.cs

```diff
@@ -28,24 +27,19 @@ public class MsalDistributedTokenCacheAdapter : MsalAbstractTokenCacheProvider
         /// <summary>
         /// Initializes a new instance of the <see cref="MsalDistributedTokenCacheAdapter"/> class.
         /// </summary>
-        /// <param name="microsoftIdentityOptions"></param>
-        /// <param name="httpContextAccessor"></param>
         /// <param name="memoryCache"></param>
         /// <param name="cacheOptions"></param>
         public MsalDistributedTokenCacheAdapter(
-                                            IOptions<MicrosoftIdentityOptions> microsoftIdentityOptions,
-                                            IHttpContextAccessor httpContextAccessor,
                                             IDistributedCache memoryCache,
-                                            IOptions<MsalDistributedTokenCacheAdapterOptions> cacheOptions)
-            : base(microsoftIdentityOptions, httpContextAccessor)
+                                            IOptions<DistributedCacheEntryOptions> cacheOptions)
```

Comment:
Why is this not `IOptions<MsalDistributedTokenCacheAdapterOptions> cacheOptions`?

Reply:
Good catch!